packages/editor/src/plugins/image/controls/upload-button.tsx

```diff
@@ -14,26 +13,27 @@ export function UploadButton({ config, state }: ImageProps) {
 
   return (
     <>
-      <label
-        className={cn(`
-          mr-2 cursor-pointer rounded-md border border-gray-500
-          px-1 text-sm transition-all focus-within:bg-editor-primary-200 hover:bg-editor-primary-200
-          focus-visible:bg-editor-primary-200
-        `)}
-      >
-        {imageStrings.upload} <FaIcon icon={faCircleArrowUp} />
-        <input
-          type="file"
-          accept="image/*"
-          className="sr-only"
-          onChange={({ target }) => {
-            if (target.files && target.files.length) {
-              void src.upload(target.files[0], config.upload)
-            }
-          }}
-          data-qa="plugin-image-upload"
-        />
-      </label>
+      <button className="z-0 mb-4 flex w-full  items-center justify-center rounded-lg bg-editor-primary-200 font-semibold text-gray-800 hover:bg-editor-primary-300 ">
+        <label tabIndex={-1} className="z-10 h-full w-full cursor-pointer py-2">
+          <span className="mr-2 inline-block">
+            <FaIcon icon={faUpload} />
+          </span>
+          <input
+            tabIndex={-1}
```

Comment:
file upload can't be opened with keyboard (chromium, macOS)

Reply:
Fixed